internal/pkg/service/buffer/store/model/model.go

```diff
@@ -18,13 +18,23 @@ const (
 )
 
 type Receiver struct {
+	*ReceiverBase
+	Exports []*Export
+}
+
+type Export struct {
+	*ExportBase
+	*Mapping
+}
+
```

Comment:
Myslim, ze nie je ziaden dovod mat tu pointre.
V pripade chyby mozes vratit prazdnu hodnotu.

Ono je to skoro jedno, ked su tie struktury male, tak si myslim, ze je lepsie uprednostnil `immutable` - hodnoty sa pri pouzivani kopiruju.

Ak chces pointre, tak potom aj `Export`, `Receiver` a upravit `store.go`, aby vracal pointre.
Teraz je to nekonzistentne.

----

Hod ma pls na review este raz, ked to rebasnes nad `main` (ked sa mergne https://github.com/keboola/keboola-as-code/pull/1023)